data/mods/Dark-Skies-Above/monsters/alien_cyborgs.json

```
@@ -286,5 +286,34 @@
     "anger_triggers": [ "FRIEND_ATTACKED", "HURT" ],
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "BASHES", "DESTROYS", "ACIDPROOF", "CAN_DIG", "ARTHROPOD_BLOOD" ]
+  },
+  {
+    "id": "mon_dks_skirmsoldier",
+    "type": "MONSTER",
+    "name": { "str": "alien skirmisher" },
+    "description": "A slouched figure, slightly smaller than an average adult human.  Beneath layers of full-body combat armor and a full suite of cybernetic modifications are glimpses of rubbery, abscessed flesh.  It is capable of short, boosted jumps using a device on its back, closing gaps to cause havoc with its wicked integrated melee weapons.  Severed organic parts dangle from its equipment, morbid trophies from past battles.",
```

```suggestion
    "description": "A slouched bipedal figure, slightly smaller than an average adult human.  Beneath layers of full-body combat armor and a full suite of cybernetic modifications are glimpses of rubbery, abscessed flesh.  It is capable of short, boosted jumps using a device on its back, closing gaps to cause havoc with its wicked integrated melee weapons.  Severed organic parts dangle from its equipment, morbid trophies from past battles.",
```

I had been envisioning these as manhacks, but creepy little cyborgs with battle trophies are great.  I just think the fact that they're bipedal needs to be made clear.

---

Bipedal is a much better word than "humanoid", which was in my original draft and removed because I felt like it was a bit too explicit.